### PR TITLE
Run `stylelint` CLI without input output help

### PR DESCRIPTION
**Pull request by evilebottnawi**
Close https://github.com/stylelint/stylelint/issues/1557

### CODE REVIEW

**Comment by jeddy3** [comment]
Won't this break stdin? e.g. 

```console
echo "a { color: pink; }" | stylelint
```

**Comment by evilebottnawi** [comment]
@jeddy3 ops, forget about stdin :smile:

**Comment by davidtheclark** [comment]
Seems ok. Have you tried out and confirm that this works with files, with stdin, with regular `stylelint --help`, and with just `stylelint`?

**Comment by evilebottnawi** [comment]
@davidtheclark yep

**Comment by davidtheclark** [comment]
@evilebottnawi, @jeddy3 Cool, we just need a changelog entry and documentation.

How about adding to changelog: 

> Added: invoking the CLI with no arguments and no `stdin` (i.e. just `stylelint`) is equivalent to `stylelint --help`.

And throw that line somewhere in the CLI docs, as well.

**Comment by jeddy3** [comment]
> we just need a changelog entry and documentation.

Sgtm.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Head
 
 - Added: `defaultSeverity` configuration option.
+- Added: invoking the CLI with no arguments and no stdin (i.e. just stylelint) is equivalent to stylelint --help.
```
Comment (davidtheclark) [inline CHANGELOG.md]: Please add ticks (\`) for code formatting.

**Comment by davidtheclark** [comment]
👍

**Pull request merged**